v1/lib/pngcreator.php

```diff
@@ -0,0 +1,53 @@
+<?php
+
+/**
+ * @file
+ * @version 0.1
+ * @copyright 2016 CN-Consult GmbH
+ * @author Max Späth <max.spaeth@cn-consult.eu>
+ */
+
+/**
+ * This class creates and saves the gamefield each round as a png file.
+ */
+class pngCreator
+{
+    /**
+     * Creates the png and saves it to lib/output/png.
```

Comment:
No! A png is no lib!!! Save it directly under /v1/tmp/png or something else (gif's should be saved aswell in this directory /tmp/gif)